src/Juno/JunoClient.ts

```diff
@@ -291,7 +300,7 @@ export class JunoClient {
   }
 
   public async favoriteNotebook(id: string): Promise<IJunoResponse<IGalleryItem>> {
-    const response = await window.fetch(`${this.getNotebooksAccountUrl()}/gallery/${id}/favorite`, {
+    const response = await window.fetch(`${this.getNotebooksUrl()}/${this.getAccount()}/gallery/${id}/favorite`, {

```

Comment:
What's the reason we don't want `${this.getNotebooksAccountUrl()}` since this `${this.getNotebooksUrl()}/${this.getAccount()}` pattern gets reused multiple times?

Reply:
Reverted to reinstate that pattern